src/pages/Transaction/TransactionCell/index.tsx

```diff
@@ -138,30 +138,35 @@ const TransactionCellIndexAddress = ({
   )
 }
 
-const isUdt = (cell: State.Cell) => cell.udtInfo && cell.udtInfo.typeHash
+const isUdt = (cell: State.Cell) => {
+  const { extraInfo } = cell
+  const udtExtraInfo = cell.cellType === 'udt' && extraInfo ? (extraInfo as State.UDTInfo) : ''
+  return udtExtraInfo && udtExtraInfo.typeHash
+}
 
 const parseNftInfo = (cell: State.Cell) => {
   if (cell.cellType === 'nrc_721_token') {
-    const nftInfo = cell.nrc721TokenInfo
+    const nftInfo = cell.extraInfo as State.Nrc721TokenInfo
     return <TransactionCellNftInfo>{`${nftInfo.symbol} #${nftInfo.amount}`}</TransactionCellNftInfo>
   }
 
   if (cell.cellType === 'm_nft_issuer') {
-    const nftInfo = cell.mNftInfo as State.NftIssuer
+    const nftInfo = cell.extraInfo as State.NftIssuer
     if (nftInfo.issuerName) {
       return sliceNftName(nftInfo.issuerName)
     }
     return i18n.t('transaction.unknown_nft')
   }
 
   if (cell.cellType === 'm_nft_class') {
-    const nftInfo = cell.mNftInfo as State.NftClass
+    const nftInfo = cell.extraInfo as State.NftClass
```

Comment:
The type assertion can be avoided by a type guard 🫣

Ref: [typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)